frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -157,15 +159,27 @@ export class ImportWizard extends Wizard {
     await isLoaded();
   }
 
-  async waitForVMWarePod() {
-    await browser.wait(until.invisibilityOf(view.vmwarePodStatusLoader), POD_CREATION_TIMEOUT_SECS);
-  }
+  async waitForSpinner() {
```

Comment:
can you please explain this function a bit?

Reply:
@suomiy  Added doc string.
The problem with the loading spinner icon is that when user selects a VM from the list, the loading spinner disappears and re-appears at least one time during the load.

And if we attempt to click next during that period, we get an error.

Reply:
I see, if it works, then okay. 

Shouldn't we also throw an exception / fail for the invalid path in case the spinner never stop spinning. We could wait for example for few minutes, so it doesn't become an infinite loop.

Reply:
Yes, that is good point, added limit of attempts.